cou/commands.py

```diff
@@ -209,6 +223,18 @@ def get_subcommand_common_opts_parser() -> argparse.ArgumentParser:
         type=purge_before_arg,
         required=False,
     )
+    subcommand_common_opts_parser.add_argument(
+        "--skip-apps",
+        dest="skip_apps",
+        choices=["vault"],
+        action=SkipAppsArgumentAction,
```

Comment:
I think we can drop the action - it doesn't bring much benefit.  If the user does add duplicated options, it won't change the logic.  Keeping it as a list also doesn't have much performance impact because the list has such a small number of items.

Reply:
Sure, that's the alternative implementation, I don't oppose change to that

Reply:
There is already the SplitArgs that transform the options passed into a set to avoid duplication that can be used. https://github.com/canonical/charmed-openstack-upgrader/blob/5f9959acdbb735ef8cddaf5bf0e9df7d8d7990c2/cou/commands.py#L80-L94

Reply:
Thanks, I don't know about this. But to me using comma separated is not as good as `--skip-apps a b c d`, it would be nice if that's for `nargs="+"` or `nargs="*"`

Reply:
Resolving this, thanks for the suggestion